pyo3-build-config/src/impl_.rs

```diff
@@ -1106,9 +1121,13 @@ impl BuildFlags {
     /// the interpreter and printing variables of interest from
     /// sysconfig.get_config_vars.
     fn from_interpreter(interpreter: impl AsRef<Path>) -> Result<Self> {
-        // sysconfig is missing all the flags on windows, so we can't actually
-        // query the interpreter directly for its build flags.
-        if cfg!(windows) {
+        let script = String::from("import sys;print(sys.version_info < (3, 13))");
+        let stdout = run_python_script(interpreter.as_ref(), &script)?;
```

Comment:
Let's please move this inside the `if cfg!(windows)` block? Otherwise we launch an extra Python interpreter run on every build :)